src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services31_invalid_tags.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        tags: ['foo']
+
+    Foo\Bar:
+        tags: invalid
+        inherit_tags: true
```

Comment:
I find this test confusing. Intuitively, I thought that tags were inherited and then validated, so the `Foo\Bar` service has the valid `tags: ['foo']` tags instead of `tags: invalid`